src/Rules/Rules.cs

```diff
@@ -43,19 +42,7 @@ public static class Rules
         /// <returns></returns>
         public static RunResult RunRuleByID(RuleId id, IA11yElement element)
```

Comment:
RuneRuleByID currently has no unit test coverage. How hard would it be to add this test?

Reply:
I wouldn't say it's "hard" so much as it provides very little value. Any change would also involve modifying how other assemblies in the solution call rules, which I am reluctant to do.

Reply:
I can see the point of "adds very little value", but since this is public, it should be trivial to add a test similar to `MonsterTest.GetTestResultsAsDictionary`, but call `Rules.RunRuleByID` instead of `Rules.RunAll`.  Another option would be to remove `Rules.RunRuleByID`, since (if memory serves), it was added for a specific scenario that is no longer in play.